jena-fuseki2/jena-fuseki-ui/src/services/mock/json-server.js

```diff
@@ -42,7 +42,7 @@ server.get('/index.html', (req, res) => res.jsonp(''))
 // GET SERVER INFO
 server.get('/\\$/server', (req, res) => {
   res.jsonp({
-    version: '4.x.y',
+    version: '5.x.y',
```

Comment:
We are going 5.x anyway, might update that in the tests too :slightly_smiling_face: